package.json

```diff
@@ -22,36 +22,35 @@
     "update:themes": "node -r dotenv/config ./scripts/update-themes.mjs"
   },
   "dependencies": {
-    "@astrojs/image": "^0.15.1",
-    "@astrojs/mdx": "^0.17.0",
+    "@astrojs/image": "^0.17.3",
+    "@astrojs/mdx": "^0.19.7",
     "@astrojs/netlify": "^2.1.3",
-    "@astrojs/prefetch": "^0.2.0",
-    "@astrojs/sitemap": "^1.1.0",
+    "@astrojs/prefetch": "^0.3.0",
+    "@astrojs/rss": "^2.4.4",
+    "@astrojs/sitemap": "^2.0.2",
     "@astrojs/solid-js": "^2.0.1",
-    "@astrojs/tailwind": "^3.0.1",
-    "@fontsource/inter": "^4.5.15",
-    "astro": "^2.0.16",
+    "@astrojs/tailwind": "^4.0.0",
+    "@fontsource-variable/inter": "^5.0.8",
+    "astro": "^2.10.9",
     "chart.js": "4.2.1",
     "clsx": "^1.2.1",
     "gray-matter": "^4.0.3",
     "node-fetch": "^3.3.0",
     "quill": "^1.3.7",
-    "sharp": "^0.31.3",
+    "sharp": "^0.32.5",
     "slugify": "^1.6.5",
     "solid-js": "^1.6.10",
-    "tailwindcss": "^3.2.4",
+    "tailwindcss": "^3.0.24",
```

Comment:
This got downgraded?

Reply:
Yeah it may not be needed, after the upgrade I ran into ridiculous build errors yelling about missing deps 3 layers deep in `autoprefixer` 😅 

This was my "just work!" fix, but let me revisit that.  It may have just been a bad lock file or local cache that was screwing with my build